src/tileset/wangset/wang_color.rs

```diff
@@ -0,0 +1,63 @@
+use std::collections::HashMap;
+
+use xml::attribute::OwnedAttribute;
+
+use crate::{
+    error::Error,
+    properties::{parse_properties, Color, Properties},
+    util::{get_attrs, parse_tag, XmlEventResult},
+    Result, TileId,
+};
+
+/// Raw data belonging to a tile.
+#[derive(Debug, PartialEq, Clone)]
+pub struct WangColor {
+    /// The name of this color.
+    pub name: String,
+    /// The color in #RRGGBB format (example: #c17d11).
```

Comment:
The "format" part is not really applicable here. Since this member is pretty self-explanatory I'd just remove the doc comment altogether.
```suggestion
    #[allow(missing_docs)]
```